setup.py

```diff
@@ -27,6 +27,7 @@
     readme = f.read()
 
 kerberos_require = ["requests_kerberos"]
+gssapi_require = ["requests_gssapi"]
```

Comment:
add it to `all_require` maybe? just for sake of completeness

Reply:
that's easy to add, but just to be clear: although those libraries do not conflict with each other, I would assume an end-user would want either one either another, but rarely both.